config/application.rb

```diff
@@ -63,10 +63,11 @@
 module Lupo
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.2
+    # config.load_defaults 5.2
```

Comment:
Do we no longer need the default configs?

I believe these are the defaults that get loaded:

3.17.2 For '5.2', new defaults from previous versions below and:

    config.active_record.cache_versioning: true
    config.action_dispatch.use_authenticated_cookie_encryption: true
    config.active_support.use_authenticated_message_encryption: true
    config.active_support.use_sha1_digests: true
    config.action_controller.default_protect_from_forgery: true
    config.action_view.form_with_generates_ids: true

3.17.3 For '5.1', new defaults from previous versions below and:

    config.assets.unknown_asset_fallback: false
    config.action_view.form_with_generates_remote_forms: true

3.17.4 For '5.0':

    config.action_controller.per_form_csrf_tokens: true
    config.action_controller.forgery_protection_origin_check: true
    ActiveSupport.to_time_preserves_timezone: true
    config.active_record.belongs_to_required_by_default: true
    config.ssl_options: { hsts: { subdomains: true } }

Reply:
I will re-enable the 5.2 default configs, although that probably doesn't make a difference.